Configurations/10-main.conf

```diff
@@ -1522,5 +1522,32 @@
         shared_extension => ".so.\$(SHLIB_MAJOR).\$(SHLIB_MINOR)",
         ranlib           => "$ENV{'RANLIB'}",
     },
+    "haiku-common" => {
+        template         => 1,
+        cc               => "cc",
+        cflags           => "",
```

Comment:
-DL_ENDIAN and -Wall are now factorized in haiku-common.
Thanks for the review.